### ISSUE TITLE
Sponsors and Backers not showing up in README.md

### ISSUE THREAD

**Issue by bryphe** [opencollective/opencollective#809]
Hi there,

Apologies if this is a silly mistake on my end - but it appears that both sponsors are not showing up on my README.md: https://github.com/onivim/oni/blob/master/README.md

And backers are not showing up correctly in my BACKERS.md:
https://github.com/onivim/oni/blob/master/BACKERS.md

In our `BACKERS.md`, it appears like our 'sponsors' are being shown via
```
 <a href="https://opencollective.com/oni#backers" target="_blank"><img src="https://opencollective.com/oni/backers.svg?width=890"></a>
```

![image](https://user-images.githubusercontent.com/13532591/34684063-bcecce74-f458-11e7-9d29-137dd8f095e2.png)

But this actually isn't correct - the first 'backer' shown there is actually our gold sponsor, so I want to make sure he's on our front page 😄  And our other backers are not shown here at all.

And in our `README.md`, we have the following:

```
<a href="https://opencollective.com/oni/sponsor/0/website" target="_blank"><img src="https://opencollective.com/oni/sponsor/0/avatar.svg"></a>
...
<a href="https://opencollective.com/oni/sponsor/9/website" target="_blank"><img src="https://opencollective.com/oni/sponsor/9/avatar.svg"></a>
```

which is always empty.

On the README.md, I experimented with some different paths, like `https://opencollective.com/oni/goldsponsor/0`, but it didn't seem to make a difference.

Perhaps related, but I notice that some collectives have a backer tier that is 'grouped': 
![image](https://user-images.githubusercontent.com/13532591/34684120-f098423a-f458-11e7-9d90-7aa19bcaf8db.png)

In other words, there is a single 'backer' tier that allows for variable order amounts - whereas in our collective, we have explicit tiers for each value amount. The 'grouped' backer tier seems much more convenient, but I didn't see in the documentation how to set that up - and perhaps that would help us correctly get the full set of 'backers'.

Thank you for your help! Let me know if there is any additional information I can give.

**Comment by piamancini** [comment]
cc/ @xdamman

**Comment by bryphe** [comment]
Thanks @piamancini ! Here's the TL;DR since my issue was pretty verbose:

- How do I enable __flexible tiers__ ?
- What is the correct markup to show my backers/bronze sponsors/silver sponsors/gold sponsors? 

The only one that seems to be working today is this one: `<a href="https://opencollective.com/oni#backers" target="_blank"><img src="https://opencollective.com/oni/backers.svg?width=890"></a>`, but it is showing my __gold sponsors__ instead of backers.

Thank you!

**Comment by xdamman** [comment]
https://opencollective.com/oni/tiers/gold-sponsor/badge.svg
![badge](https://opencollective.com/oni/tiers/gold-sponsor/badge.svg)
https://opencollective.com/oni/tiers/gold-sponsor/0/avatar.png
![avatar](https://opencollective.com/oni/tiers/gold-sponsor/0/avatar.png)
https://opencollective.com/oni/tiers/gold-sponsor/0/website

Same with `silver-sponsor`, `bronze-sponsor`

For flexible tiers, we haven't exposed an interface to edit them yet, so you need to contact support to update them for you.

**Comment by xdamman** [comment]
For more options see 
https://github.com/opencollective/frontend/blob/master/src/server/routes.js#L63-L76

and also https://opencollective.com/widgets

**Comment by bryphe** [comment]
Excellent, thanks @xdamman for your help! The `gold-sponsor`, `silver-sponsor`, and `bronze-sponsor` links are working great.

How do I show my 'backer' tiers? I tried `https://opencollective.com/oni/tiers/backer/0/avatar.png`, but only `0` works, and it only shows the empty user:
![avatar]( https://opencollective.com/oni/tiers/backer/0/avatar.png)

> For flexible tiers, we haven't exposed an interface to edit them yet, so you need to contact support to update them for you.

Ok, I'll give that a shot... and perhaps if I have a flexible tier for backers, it will also address the issue I have above (URL route to get all backers images). It'd be great if I had a flexible 'backer' tier with just a route like `https://opencollective.com/oni/tiers/backer/0/avatar.png` to get the data.

**Comment by xdamman** [comment]
That's because you only have one person in the backer tier (and someone that doesn't have a custom avatar). Other indexes will return a 1px transparent png (this is meant for README on Github). They will be replaced by the avatar of new backers as they come in.

I went ahead and consolidated all your backer tiers into one single tier with multiple amount presets.

**Comment by bryphe** [comment]
> I went ahead and consolidated all your backer tiers into one single tier with multiple amount presets.

Awesome, thank you @xdamman ! Looks great.

**Comment by bryphe** [comment]
I'll close this out. Thanks again for the help.
